package-lock.json

```diff
@@ -0,0 +1,1366 @@
+{
```

Comment:
having this package-lock.json should also sort out dependabot security alert : https://github.com/OWASP/threat-dragon/security/dependabot/td.site/package.json/pug/open